src/entity/base_entity.rs

```diff
@@ -4,13 +4,12 @@ use crate::{
     RelationTrait, RelationType, Select, Update, UpdateMany, UpdateOne,
 };
 use sea_query::{Alias, Iden, IntoIden, IntoTableRef, IntoValueTuple, TableRef};
-use std::fmt::Debug;
 pub use strum::IntoEnumIterator as Iterable;
 
 /// Ensure the identifier for an Entity can be converted to a static str
-pub trait IdenStatic: Iden + Copy + Debug + 'static {
+pub trait IdenStatic: Iden + Copy + 'static {
     /// Method to call to get the static string identity
-    fn as_str(&self) -> &str;
+    fn as_str(&self) -> &'static str;
```

Comment:
If we make this returns static reference, it'd be a massive breaking change https://github.com/SeaQL/sea-orm/pull/1267/commits/3cca57f5ac9ea37808ea807bd144893022e6e7cf

Reply:
I think not. I don't think there is a way to get a non-static reference to a string given that the struct must itself is `Copy`

Reply:
Only if someone is using an `ArrayString` inside a custom `Entity` definition.

Reply:
> Only if someone is using an ArrayString inside a custom Entity definition.

Which I don't think someone had actually done that. So I am okay to this breaking change.